Building with internal fmt, only version 3.0.1 builds successfully

Kodi's [build deps](http://mirrors.kodi.tv/build-deps/sources/?C=N&O=A) include fmt versions 3.0.1, 4.1.0, and 5.1.0, but I am only able to build 18.0b5 for Linux (x86_64) using fmt-3.0.1 when I build using these internal libs.
```...
[100%] Linking CXX static library libkodi.a
[100%] Built target libkodi
Scanning dependencies of target kodi
make[2]: *** No rule to make target 'build/lib/libfmt.a', needed by 'kodi-x11'.  Stop.
make[2]: *** Waiting for unfinished jobs....
[100%] Building CXX object CMakeFiles/kodi.dir/xbmc/platform/posix/main.cpp.o
make[1]: *** [CMakeFiles/Makefile2:1132: CMakeFiles/kodi.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
```
Link to complete build log: https://gist.github.com/graysky2/18b84442fc7f19802dcf07ff5ace4e5e

Here is my cmake stanza:
```
cmake -DCMAKE_INSTALL_PREFIX=/usr \
-DCMAKE_INSTALL_LIBDIR=/usr/lib \
-DENABLE_EVENTCLIENTS=ON \
-DENABLE_INTERNAL_FFMPEG=ON \
-DENABLE_INTERNAL_FMT=ON \
-DENABLE_INTERNAL_CROSSGUID=ON \
-DENABLE_INTERNAL_FSTRCMP=ON \
-DENABLE_INTERNAL_FLATBUFFERS=ON \
-Dlibdvdcss_URL="/libdvdcss-1.4.2-Leia-Beta-5.tar.gz" \
-Dlibdvdnav_URL="/libdvdnav-6.0.0-Leia-Alpha-3.tar.gz" \
-Dlibdvdread_URL="/libdvdread-6.0.0-Leia-Alpha-3.tar.gz" \
-DFFMPEG_URL="/ffmpeg-4.0.3-Leia-Beta5.tar.gz" \
-DFMT_URL="/fmt-5.1.0.tar.gz" \
-DCROSSGUID_URL="/crossguid-8f399e8bd4.tar.gz" \
-DFSTRCMP_URL="/fstrcmp-0.7.D001.tar.gz" \
-DFLATBUFFERS_URL="/flatbuffers-1.9.0.tar.gz" \
-DENABLE_AIRTUNES=OFF \
-DENABLE_AVAHI=OFF \
-DENABLE_BLUETOOTH=OFF \
-DENABLE_CEC=OFF \
-DENABLE_OPTICAL=OFF \
-DENABLE_SMBCLIENT=OFF \
-DENABLE_SSH=OFF \
-DENABLE_UPNP=OFF \
-DENABLE_VDPAU=OFF \
../"xbmc-18.0b5-Leia"
```

Interestingly, if I remove the cmake options to use the internal fmt, and if I use fmt v5.1.0 in my build chroot, the build completes without errors.

Note that I tried building with the internal fmt 5.1.0 with https://github.com/xbmc/xbmc/pull/14797 applied but got the same result.

## Expected Behavior
Successful build using the internal fmt v5.1.0.

## Actual Behavior
Build fails as detailed above.

## Possible Fix
Unknown to me.


### Your Environment
Used Operating system:
 - [X] Linux

 - Operating system version/name: Arch x86_64
 - Kodi version: 18.0b5

--- a/cmake/modules/FindFmt.cmake
+++ b/cmake/modules/FindFmt.cmake
@@ -40,6 +40,7 @@ if(ENABLE_INTERNAL_FMT)
                       PREFIX ${CORE_BUILD_DIR}/fmt
                       CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}
                                  -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}
+                                 -DCMAKE_INSTALL_LIBDIR=lib
                                  -DFMT_DOC=OFF
                                  -DFMT_TEST=OFF
                                  "${EXTRA_ARGS}"

